AK/SIMD.h

```diff
@@ -15,11 +15,13 @@ using i8x4 = i8 __attribute__((vector_size(4)));
 using i8x8 = i8 __attribute__((vector_size(8)));
 using i8x16 = i8 __attribute__((vector_size(16)));
 using i8x32 = i8 __attribute__((vector_size(32)));
+using i8x64 = i8 __attribute__((vector_size(64)));
```

Comment:
Wait these are 512 bits in size....
So they are thechnically AVX512

Reply:
Didn't realise that. I'll get rid of it.